packages/zosmf/src/CheckStatus.ts

```diff
@@ -36,6 +36,10 @@ export class CheckStatus {
         return ZosmfRestClient.getExpectJSON(session, infoEndpoint);
     }
 
+    public static async isZosVersionGreaterThan(session: AbstractSession, version: string): Promise<boolean> {
```

Comment:
Thanks for adding this! We might also be able to leverage this in Zowe Explorer for some pre-existing features 😀 

_Reviewer note:_ At first I was thinking that `version` could be more strongly typed - it would prevent anyone from calling this function with a non-existent or unsupported version, but it would also limit this function's usability to versions that the SDK supports, and would require updating it for each z/OSMF release. I thought of a type that would grab all the version numbers from the string, but due to the caveats above, feel free to disregard. I just figured I'd post it in case anyone found it interesting 😋 

```typescript
type ZosmfVersion = (typeof ZosmfConstants.VERSIONS)[keyof typeof ZosmfConstants.VERSIONS];
public static async isZosVersionGreaterThan(session: AbstractSession, version: ZosmfVersion): Promise<boolean>;
```